CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 **Note:** This is a cumulative changelog that outlines all of the Apollo Link project child package changes that were bundled into a release on a specific day.
 
+## 2019-05-13
```

Comment:
Could you just move this under ## vNext ###apollo-link 👍  thanks for already doing this

Reply:
Sure thing!